app/controllers/user_sessions_controller.rb

```diff
@@ -41,17 +41,16 @@ def create
               message = ts("The password you entered has expired. Please click the 'Reset password' link below.")
             end
           elsif user.active?
-            if @user_session.being_brute_force_protected? 
-           
+            if @user_session.being_brute_force_protected?
               message = ts("Your account has been locked for 5 minutes due to too many failed login attempts.")
             else
-              message = ts("The password or user name you entered doesn't match our records. Please try again or follow the 'Forgot password?' link below.  If you still can't log in, please visit <a href=\"http://archiveofourown.org/admin_posts/1277\">Problems When Logging In</a> for help.".html_safe)
+              message = ts("The password or user name you entered doesn't match our records. Please try again or <a href=\"http://archiveofourown.org/passwords/new\">reset your password</a>.".html_safe)
```

Comment:
Whoops, sorry, I think the issue was unclear -- we just want to change the second sentence, but the third (about problems when logging in) can stay.

Also, we're going to be changing over to https sooner rather than later (fingers crossed!) so we don't want to include the `http://` bit in the URL. A better way to do this might be: `<a href=\"#{new_password_path}\">reset you password</a>` If you could change the other link to work like this, too, that would be great! It would probably be something like `<a href=\"#{admin_posts_path + "/1277"}\">link text</a>` (I haven't tried it and am just guessing at where all those quotation marks, slashes, and braces go, so you'll have to fiddle with it a bit).